specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3500,54 +3532,72 @@
           }
         ],
         "responses": {
```

Comment:
This will always return a 204 with no response body? I would assume the "200" response is missing, and "default"

Reply:
These new APIs (WebApps_ListSyncStatus, WebApps_SyncFunctions) are just newly added aliases for the existing APIs, and ultimately map to the same implementation as the existing WebApps_SyncFunctionTriggers operation, and return the same response. As you can see, for the existing WebApps_SyncFunctionTriggers it returns this same 204 response. Not sure why the existing APIs are missing "default" blocks.